modules/atomic-widgets/prop-types/prop-type.php

```diff
@@ -20,9 +42,10 @@ public function get_dynamic_categories(): array {
 	}
 
 	public function jsonSerialize(): array {
-		return [
-			'type' => $this->get_type(),
+		return array_merge( [
+			'key' => $this->get_key(),
 			'dynamic_categories' => $this->get_dynamic_categories(),
-		];
+			'default' => $this->default,
+		], $this->settings );
```

Comment:
the dynamic categories will now be passed to the editor on each prop in the schema, which means there will be duplicates. Do we care about it?

Reply:
Can we do it differently? just pass some configuration on the "dynamic module" (we do not have a module for dynamic but we can create a class that is responsible for it) also the category should not be hardcoded in the prop type